script/upgrades/MU03/MU03Checks.sol

```diff
@@ -0,0 +1,687 @@
+        );
+        revert("asset0 of pool does not match the expected asset0. See logs.");
+      }
+
+      // verify asset1 of the deployed pool against the config
+      if (pool.asset1 != poolConfigs[i].asset1) {
+        console2.log(
+          "The asset1 of deployed pool: %s does not match the expected asset1: %s.",
+          pool.asset1,
+          poolConfigs[i].asset1
+        );
+        revert("asset1 of pool does not match the expected asset1. See logs.");
+      }
+
+      // verify asset0 is always a stable asset
+      require(
+        pool.asset0 == cUSD || pool.asset0 == cEUR || pool.asset0 == cBRL,
+        "asset0 is not a stable asset in the exchange"
+      );
+      // verify asset1 is always a collateral asset
+      require(
+        pool.asset1 == celoToken || pool.asset1 == bridgedUSDC,
+        "asset1 is not CELO or bridgedUSDC in the exchange"
+      );
+    }
+    console2.log("\tExchanges correctly configured 🤘🏼");
+  }
+
+  function verifyTradingLimits() public view {
+    IBrokerWithCasts _broker = IBrokerWithCasts(address(broker));
+    BiPoolManager bpm = getBiPoolManager();
+    bytes32[] memory exchanges = bpm.getExchangeIds();
+
+    for (uint256 i = 0; i < exchanges.length; i++) {
+      bytes32 exchangeId = exchanges[i];
+      IBiPoolManager.PoolExchange memory pool = bpm.getPoolExchange(exchangeId);
+      bytes32 limitId = exchangeId ^ bytes32(uint256(uint160(pool.asset0)));
+      TradingLimits.Config memory limits = _broker.tradingLimitsConfig(limitId);
+
+      if (limits.timestep0 == 0 || limits.timestep1 == 0 || limits.limit0 == 0 || limits.limit1 == 0) {
+        console2.log("The trading limit for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were set.");
+      }
+      // verify configured trading limits for all pools
+      if (poolConfigs[i].asset0_limit0 != limits.limit0) {
+        console2.log("limit0 for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were configured correctly.");
+      }
+      if (poolConfigs[i].asset0_limit1 != limits.limit1) {
+        console2.log("limit1 for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were configured correctly.");
+      }
+      if (poolConfigs[i].asset0_limitGlobal != limits.limitGlobal) {
+        console2.log("limitGlobal for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were configured correctly.");
+      }
+      if (poolConfigs[i].asset0_timeStep0 != limits.timestep0) {
+        console2.log("timestep0 for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were configured correctly.");
+      }
+      if (poolConfigs[i].asset0_timeStep1 != limits.timestep1) {
+        console2.log("timestep1 for %s, %s was not set ❌", pool.asset0, pool.asset1);
+        revert("Not all trading limits were configured correctly.");
+      }
```

Comment:
Let's also check the `flags` here, to make sure the limits are enabled, values can be set there but if the flag isn't correct it won't matter.